src/App.java

```diff
@@ -1,109 +1,74 @@
-import java.util.Scanner;
+
 
 public class App {
     
     public static void main(String[] args) throws Exception {
-        firstTask();
-        secondTask();
-        thirdTask();
-        fourTask();
-        fiveTask();
-        sixTask();
-        sevenTask();
-        
-    }
-    public static void firstTask(){
-        Scanner scan = new Scanner(System.in);
-        System.out.println("How old are you?");
-        String ageString = scan.nextLine();
-        int age = Integer.parseInt(ageString);
-        if(!ageString.isEmpty() && age > 0){
-            if(age >= 18){
-                System.out.println("you are sooo adult \n" + age + " it's to much");
+        //firstTask();
+        int[] arr0 = new int[3];
+        arr0[0] = 1;
+        arr0[1] = 2;
+        arr0[2] = 3;
+        double[] arr1= {1.57, 7.654, 9.986};
+        char[] name = {'V','l','a','d'};
+        //secondTask();
+        for(int i=0;i<arr0.length;i++){
+            if(i!=2){
```

Comment:
Не нужно хардкодить индекс последнего элемента. Его нужно взять из свойства length
```suggestion
            if(i!=arr0.length-1){
```